router/router.go

```diff
@@ -316,6 +316,16 @@ func (worker *workerT) handleWorkerDestinationJobs() {
 
 			routerResponseStat := stats.GetRouterStat("router_response_counts", stats.CountType, worker.rt.destName, respStatusCode)
 			routerResponseStat.Count(len(destinationJob.JobMetadataArray))
+			destinationTag := misc.GetTagName(destinationJob.Destination.ID, destinationJob.Destination.Name)
+			tags := map[string]string{
+				"module":      "router",
+				"destType":    worker.rt.destName,
+				"destination": destinationTag,
+			}
+			eventsDeliveredStat := stats.NewTaggedStat("event_delivery", stats.CountType, tags)
```

Comment:
should we create NewTaggesStat for every job or have them in a map by tagName and use that?